service.toml

```diff
@@ -1,11 +1,13 @@
 name = "storj"
 
 [namespace.storage]
+implement = ["copier", "mover", "direr"]
```

Comment:
How remove all these `implement` so that we can focus on implementing `Storager`?